scripts/overview.json

```diff
@@ -3612,6 +3650,7 @@
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
           "decimals": 1,
+          "description": "the data amount on each TiKV node",
```

Comment:
The data size?

Reply:
@zhangjinpeng1987 What's the difference between the engine size here and the cluster data size above?